auction-app-backend/src/main/java/com/atlantbh/auctionappbackend/SampleEntity/SampleEntity.java

```diff
@@ -0,0 +1,57 @@
+package com.atlantbh.auctionappbackend.SampleEntity;
```

Comment:
I suggest removing `SampleEntity` folder and creating simple health check API which should be exposed through `HealthCheckController`. This API could return only `OK` just to ensure that service is up. This way we can also check that Swagger UI is also up and works as expected (health check API should be shown on the Swagger UI).

Reply:
I agree. I will work on that.